elements/pfe-card/pfe-card.scss

```diff
@@ -1,278 +1,75 @@
-    ),
-
-    footer: (
-      spacing--horizontal: calc(#{pfe-var(container-spacer)} / 2)
-    ),
-  ),
-);
-
-// Nested internal variables (pfe-local calls), maps cannot "self-reference"
-@include merge-local-variables((
-  // Internal spacing; elements inside the card (header, body, footer regions)
-  spacing--vertical: pfe-local(spacing),
-
-  // Combine above variables or allow single override point via variable
-  Padding: pfe-local(PaddingTop) pfe-local(PaddingRight) pfe-local(PaddingBottom) pfe-local(PaddingLeft),
-
-  //-- Border variable encompasses width, style, and color
-  Border: pfe-local(BorderWidth, 0) pfe-local(BorderStyle, solid) pfe-local(BorderColor, pfe-var(surface--border))
-));
-
-$size-small: (
-  PaddingTop:     pfe-var(container-spacer),
-  PaddingRight:   pfe-var(container-spacer),
-  PaddingBottom:  pfe-var(container-spacer),
-  PaddingLeft:    pfe-var(container-spacer)
-);
-
 :host {
-  --context: #{pfe-local(context)};
-
-  --pfe-theme--color--surface--lightest: #ffffff;
-  --pfe-theme--color--surface--lighter: #ececec;
-  --pfe-theme--color--surface--base: #f0f0f0;
-  --pfe-theme--color--surface--darker: #3c3f42;
-  --pfe-theme--color--surface--darkest: #151515;
-  --pfe-theme--color--surface--accent: #004080;
-  --pfe-theme--color--surface--complement: #002952;
-
-  // Start of style declarations for host element
-  display: flex;
-  flex-direction: column;
-  justify-items: flex-start;
-  // This allows the card to fill it's container if necessary
-  align-self: stretch;
-
-  padding:       pfe-local(Padding);
-  border:        pfe-local(Border);  // @TODO add automatic border when lightest card is on lightest background?
-  border-radius: pfe-local(BorderRadius);
-
-  // This property ensures that children in the slots do no overflow
-  // the border-radius being set on the container
-  overflow: hidden;
-
-  // Base colors
-  background-color:    var(--pfe-card--BackgroundColor, var(--pfe-context-background-color));
-  background-position: pfe-local(BackgroundPosition);
-  color: pfe-broadcasted(text);
-
-  // Remove background color for print
-  @include pfe-no-print-background;
-
-  // Add the border to the card for print
-  @include pfe-print-media {
-    border-radius: pfe-fetch(surface--border-radius);
-    border: pfe-fetch(surface--border-width) pfe-fetch(surface--border-style) pfe-fetch(surface--border);
-  }
+  background-color: var(--pf-c-card--BackgroundColor, var(--pf-global--BackgroundColor--100, #ffffff));
+  box-shadow: var(--pf-c-card--BoxShadow, var(--pf-global--BoxShadow--sm, 0 0.0625rem 0.125rem 0 rgba(3, 3, 3, 0.12), 0 0 0.125rem 0 rgba(3, 3, 3, 0.06)));
 }
 
-/* @deprecated */
-:host([color]),
-:host([color-palette]),
-:host([on]) {
-  background-color:    var(--pfe-card--BackgroundColor, var(--pfe-context-background-color));
+:host([size="compact"]) {
+  --_pf-c-card__body--FontSize: var(--pf-c-card--size-compact__body--FontSize, var(--pf-global--FontSize--sm, .875rem));
```

Comment:
Yes they're ours, I can simplify them since they are pretty long though I thought we were keeping the variable styles outside of BaseCard?  These are also not relevant for rh-card since they go along with the variants present in pfe-card if that makes sense?

Reply:
what's important in Base* styles isn't that they don't have variables, but that they don't have patternfly-specific variables, but 'private' props that are related to functionality are ok, as long as they don't reference pfe defaults. if they use pfe defaults, keep them here.